src/deprecations.json

```diff
@@ -0,0 +1,20 @@
+{
+  "cssCustomProperties": [
+    "--nds-black",
+    "--nds-grey",
+    "--nds-medium-grey",
+    "--nds-lightest-grey",
+    "--nds-smoke-grey",
+    "--nds-red",
+    "--nds-white",
+    "--nds-font-family",
+    "nds-header-font",
+    "subdued-20-opacity",
+    "subdued-10-opacity"
+  ],
+  "components": [
+    "Card",
+    "Dropdown",
+    "PlainButton"
+  ]
+}
```

Comment:
In the future, we could automate the creation of this file in `dist/` using jsDoc (e.g. the `@deprecated` tag across all js/jsx/scss files).

Worst case scenario, this file goes stale. That seems like a tolerable risk, as this is an opt-in file from `dist` and will not be included in the functional library.